place_tracker/lib/place_map.dart

```diff
@@ -412,15 +387,17 @@ class _PlaceMapState extends State<PlaceMap> {
       maxLong = max(maxLong, place.longitude);
     }
 
-    await controller.animateCamera(
-      CameraUpdate.newLatLngBounds(
-        LatLngBounds(
-          southwest: LatLng(minLat, minLong),
-          northeast: LatLng(maxLat, maxLong),
+    WidgetsBinding.instance.addPostFrameCallback((_) {
```

Comment:
Necessary to make it work, as we are called from an asynchronous listener

Reply:
Could we add a comment here?